src/pages/CollectionSingleViewPage.tsx

```diff
@@ -63,7 +64,11 @@ export const CollectionSingleViewPage = () => {
       highlightedOnly: queryParams.get('highlightedOnly') === 'true',
     };
   }, [queryString]);
-  const dateRangeSelector = new SpecialDateRangeSelector('Past6M'); // TODO
+
+  // Date range
+  const [dateRangeSelector, setDateRangeSelector] = useState<DateRangeSelector | SpecialDateRangeSelector>(
```

Comment:
`SpecialDateRangeSelector` is a subtype of `DateRangeSelector`, so there should be no need to mention both.